src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdNotifier.cs

```diff
@@ -48,7 +46,7 @@ public void Notify(ServiceState state)
 
         private static string GetNotifySocketPath()
         {
-            string socketPath = Environment.GetEnvironmentVariable(NOTIFY_SOCKET);
+            string socketPath = Environment.GetEnvironmentVariable(SystemdHelpers.NOTIFY_SOCKET_ENVVAR_KEY);
```

Comment:
I'd keep the `private const string` in this class.

Reply:
Actioned